### PR TITLE
Adds learn link to the website

### PR DESCRIPTION
**Pull request by codefromthecrypt**

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
slight glitch as https://deploy-preview-421--func-e.netlify.app/learn/ doesn't show up..

**Comment by codefromthecrypt** [comment]
A little stumped as locally it works, but the netlify one is 404

Screen Shot 2022-01-05 at 2 41 56 PM

File: Makefile [hunk 1]
```diff
@@ -248,7 +248,7 @@ check: ## Verify contents of last commit
 
 site: ## Serve website content
 	@git submodule update
-	@cd site && $(go) run $(hugo) server --disableFastRender -D
+	cd site && $(go) run $(hugo) server --minify --disableFastRender --baseURL localhost:1313 --cleanDestinationDir -D
```
Comment (codefromthecrypt) [inline Makefile:251]: when this works, put the '@' back

**Comment by codefromthecrypt** [comment]
ps netlify == https://app.netlify.com/sites/func-e wrt config and build status anyway

**Comment by codefromthecrypt** [comment]
note this is still busted and still waiting for help

**Comment by aditi-Kh** [comment]
@pmerrison @codefromthecrypt - The Envoy Fundamentals launch is happening tomorrow(Feb4). Do we have a solutions to this yet?

CC: @AswinBehera - Can you help debug?

**Comment by AswinBehera** [comment]
@codefromthecrypt I am getting 404 on clicking the "Start Learning" on the Navigation bar on my local server.  

All the other links on the navigation bar point to other domains, and in the "Start Learning" case, it's within the func-e domain. Maybe the inter-page navigation on func-e has not been configured properly.

**Comment by codefromthecrypt** [comment]
Right, this is what's needed to be figured out. `make site` should ensure you are using the right Hugo version also.

good luck and thanks for helping on this @AswinBehera

**Comment by lunchboxav** [comment]
alright, founded the culprit. the theme seems to be confused when generating pages because it doesn't recognize the default index page. This works on my local, so hopefully it'll work on netlify

**Comment by lunchboxav** [comment]
Yeap. it works now https://deploy-preview-421--func-e.netlify.app/learn/ cc: @AswinBehera @codefromthecrypt. It's yours to merge now :)

File: Makefile (hunk 1 above)
Comment (codefromthecrypt) [inline Makefile:251]:
```suggestion
	@cd site && $(go) run $(hugo) server --minify --disableFastRender --baseURL localhost:1313 --cleanDestinationDir -D
```

File: site/content/_index.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 +++
 title = "func-e makes running Envoy® easy"
+layout = "single"
```
Comment (codefromthecrypt) [inline site/content/_index.md:3]: this was needed right? just for my knowledge

**Comment by codefromthecrypt** [comment]
looks great @lunchboxav! 🙇

**Review by codefromthecrypt — COMMENTED** [review]
thanks for this. The windows fail is unrelated

File: site/content/_index.md (hunk 2 above)
Comment (lunchboxav) [reply site/content/_index.md:3]: Yes, correct @codefromthecrypt. Without it, the content won't be assigned the correct class, hence breaking the presentation. But it's not required for the additional pages. Seems like the theme doesn't automatically assign single page layout for index page.

**Pull request merged**